src/context/WalletProvider/NativeWallet/components/NativeStart.tsx

```diff
@@ -73,6 +75,25 @@ export const NativeStart = ({ history }: RouteComponentProps) => {
           >
             <Text translation={'walletProvider.shapeShift.start.import'} />
           </Button>
+          <Divider mt={4} />
```

Comment:
Once the feature flag PR is merged, use the feature flag to hide this unless the feature is on.